web/functions/cors/README.md

```diff
@@ -0,0 +1,14 @@
+# CORS Function
+
+Netlify function to set up CORS for API requests.
+
+## Usage
+
+First, the spreadsheet must be shared with the service account `hodp-netlify@hodp-sheets.iam.gserviceaccount.com`. Then, to use in code:
```

Comment:
Clarify whether the spreadsheet needs to be public, or whether it only needs to be shared with `hodp-netlify@hodp-sheets.iam.gserviceaccount.com`. Also, just made a [folder](https://drive.google.com/drive/u/0/folders/1ByD7GreoggyWpL-gpEgVRymVR_83XlcH) to consolidate our Google Spreadsheet data.

Reply:
Great!